### PR TITLE
[SPARK-25431][SQL][EXAMPLES] Fix function examples and unify the format of the example results.

### PR DESCRIPTION
**Pull request by ueshin**
## What changes were proposed in this pull request?

There are some mistakes in examples of newly added functions. Also the format of the example results are not unified. We should fix and unify them.

## How was this patch tested?

Manually executed the examples.

### CODE REVIEW

**Review by gatorsmile — COMMENTED** [review]
LGTM

Thanks! Merged to master/2.4

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala [hunk 1]
```diff
@@ -131,7 +131,7 @@ case class Size(child: Expression) extends UnaryExpression with ExpectsInputType
   examples = """
     Examples:
       > SELECT _FUNC_(map(1, 'a', 2, 'b'));
-       [1,2]
+       [1, 2]
```
Comment (dongjoon-hyun) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala]: ?

**Comment by dongjoon-hyun** [comment]
@ueshin and @gatorsmile .

Sorry, but could you explain where did you get the new result? IIRC, historically, the example illustrates the result from `spark-sql` like Hive description. The new change doesn't consistent to the master result.

**Comment by dongjoon-hyun** [comment]
If you don't mind, please revert this.
cc @cloud-fan

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala [hunk 2]
```diff
@@ -718,7 +718,7 @@ case class MapConcat(children: Seq[Expression]) extends ComplexTypeMergingExpres
   examples = """
     Examples:
       > SELECT _FUNC_(array(struct(1, 'a'), struct(2, 'b')));
-       {1:"a",2:"b"}
+       [1 -> "a", 2 -> "b"]
```
Comment (dongjoon-hyun) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala]: The previous one is correct.
```
spark-sql> SELECT map_from_entries(array(struct(1, 'a'), struct(2, 'b')));
{1:"a",2:"b"}
```

**Comment by ueshin** [comment]
@dongjoon-hyun I picked the string from `sql(...).show(truncate = false)` and modified a little. I'm sorry but I didn't notice that we use the `spark-sql` to pick the result. Let me send a follow-up pr.

**Comment by dongjoon-hyun** [comment]
@ueshin . Could you revert the current one and send a new one please?

**Comment by ueshin** [comment]
@dongjoon-hyun Sure, reverted. Thanks!

**Comment by dongjoon-hyun** [comment]
Thank you, @ueshin .

**Pull request closed without merging**